docs/guide/machine-learning.asciidoc

```diff
@@ -29,3 +33,48 @@ model in Elasticsearch
 >>> es_model.predict(training_data[0])
 [0 1 1 0 1 0 0 0 1 0]
 ------------------------
+
+[discrete]
+[[ml-nlp-pytorch]]
+=== Natural language processing (NLP) with PyTorch
+
+For NLP tasks, Eland allows importing PyTorch trained BERT models into {es}. 
+Models can be either plain PyTorch models, or supported 
+https://huggingface.co/transformers[transformers] models from the
+https://huggingface.co/models[Hugging Face model hub].
+
+[source,bash]
+------------------------
+$ eland_import_hub_model \
+  --url http://localhost:9200/ \
+  --hub-model-id elastic/distilbert-base-cased-finetuned-conll03-english \
+  --task-type ner \
+  --start
+------------------------
+
```

Comment:
Maybe it's obvious to folks who use eland a lot, but as a novice user, I didn't understand the purpose of these two examples, which seem to me to accomplish the same thing. It might be nice to add a little context in an intro to these snippets.